### DISCUSSION TITLE
[SUGGESTION] Move the new ha-wican repository to the meatpiHQ organization or to a new shared organization for all WiCAN projects on GitHub?

### DISCUSSION THREAD

**Discussion by Hedda** [meatpiHQ/wican-fw#258] [Ideas]
@jay-oswald @meatpiHQ can I suggest that you consider asking/discussing the possibility of having your new ha-wican repository moved from https://github.com/jay-oswald/ha-wican/issues/new moved to the meatpiHQ organization @ https://github.com/meatpiHQ on GitHub or alternativly to new shared organization for all WiCAN projects on GitHub?

@meatpiHQ Perhaps best would to breakout all repos specific to WiCAN into their own originazation so not confused with your other products and projects?

* https://github.com/meatpiHQ

Or new org as

https://github.com/WiCAN

Anyway, the point is that if this is suppose to become an official part of the WiCAN project then the upstream/mainline copy of this repo should be under the same orignation on GitHub to really become upstream, with shared access per repo by the collective of developers that make up the organization? 

Note that "organization" is just GitHub's terminology as I understand it is more an informal collaboration between like-minded WiCAN developers in order to collect all WiCAN related projects ín one place and shared access rights, (noting that a WiCAN organization on GitHub does not have to have public members and that you must be a member to see who’s a part of that organization, and that access can still be given per repository).

All WiCAN related repositories can then later be moved there over time as a way for like-minded WiCAN developers that are members of the WiCAN organization with yourself included can have shared ownership and access to your repositories and the other projects for collaboration within that community? 

Moving it could potentially also help new WiCAN developers take notice of subprojects like ha-wican.

Hope that you at least will consider this option and if you think it is a good idea then hoping that you all could come to an agreement that will make everyone happy with you continuing this ha-wican repository under the same ”WiCAN” or ”meatpiHQ” organization together without getting offended by any ownership conflicts.

**Comment by jay-oswald** [comment]
Theres really no benefit to this? Maybe a new org at some point, but for now meatpi is main dev for FW, and I'm main dev for the HA integration. This repo is linked from the official docs,and has its docs as part of the official ones, I don't think its confusing.

If it does grow to be bigger with more people involved than a dedicated Org does make the most sense its just not at the point where its worth stuffing around.
